src/tasks/copy-template-files.ts

```diff
@@ -276,19 +281,19 @@ const setUpExternalExtensionFiles = async (options: Options, tmpDir: string) =>
 
 export async function copyTemplateFiles(options: Options, templateDir: string, targetDir: string) {
   copyOrLink = options.dev ? link : copy;
-  const basePath = path.join(templateDir, baseDir);
+  const basePath = path.join(templateDir, BASE_DIR);
   const tmpDir = path.join(targetDir, EXTERNAL_EXTENSION_TMP_FOLDER);
 
   // 1. Copy base template to target directory
   await copyBaseFiles(basePath, targetDir, options);
 
-  // 2. Copy extensions folders
-  await Promise.all(
-    options.extensions.map(async extension => {
-      const extensionPath = extensionDict[extension].path;
-      await copyExtensionsFiles(options, extensionPath, targetDir);
-    }),
-  );
+  let solidityFrameworkPath;
```

Comment:
[5543001](https://github.com/scaffold-eth/create-eth/pull/68/commits/5543001509c032f89628dd63bafbf3441e9973c9) now it's `const`, but still out of `if` since it's used later